e2e/testcases/stress_test.go

```diff
@@ -881,13 +881,14 @@ func truncateSourceErrors() testpredicates.Predicate {
 	}
 }
 
-func validateNumberOfObjectsEquals(nt *nomostest.NT, gvk schema.GroupVersionKind, count int, opts ...client.ListOption) {
+func validateNumberOfObjectsEquals(nt *nomostest.NT, gvk schema.GroupVersionKind, count int, opts ...client.ListOption) error {
```

Comment:
Care to pull out this refactor as a standalone change?

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/pull/1385